docs/content/en/blog/news/control-plane-on-ecs.md

```diff
@@ -0,0 +1,285 @@
+---
+date: 2022-02-07
+title: "PipeCD best practice 02 - control plane on ECS"
+linkTitle: "PipeCD best practice 02"
+weight: 996
+description: "This blog is a part of PipeCD best practice series, a guideline for you to operate your own PipeCD on Amazon ECS."
+author: Yohei Namba ([@kevin55156](https://twitter.com/kevin55156))
+---
+
+This blog is a part of PipeCD best practice series, a guideline for you to operate your own PipeCD.
+Currently, you can deploy and operate the PipeCD control plane on a Kubernetes cluster easily, but some developers that would like to introduce PipeCD can not prepare Kubernetes environments. If you have the same problem, this blog is for you. We will show you how to deploy the PipeCD control plane on Amazon ECS.
+
+### Architecture
+> Note: Please refer to [architecture-overview](docs/user-guide/managing-controlplane/architecture-overview/) docs for definitions of PipeCD components such as server, ops, cache, datastore and filestore.
+
+![](/images/control-plane-on-ecs.png)
+
+Following the above graph for PipeCD control plane runs on Amazon ECS, we have to prepare these next components
+
+### Secrets Manager
+You should put config files (control-plane-config.yaml, envoy-config.yaml) on Secrets Manager because config files contain some credentials such as database passwords.
+The examples of config files for ECS are [here](https://github.com/pipe-cd/control-plane-aws-ecs-terraform-demo/tree/main/config). Please edit these files according to your environment.
+```
+aws secretsmanager create-secret --name control-plane-config \
+--description "Configuration of control plane" \
+--secret-string `base64 control-plane-config.yaml`
+aws secretsmanager create-secret --name envoy-config \
+--description "Configuration of control plane" \
+--secret-string `base64 enovy-config.yaml`
+```
+You should also put encryption key
+```
+aws secretsmanager create-secret --name encryption-key \
+--description "Encryption key for control plane" \
+--secret-string `openssl rand 64 | base64`
+```
+
+### RDS(datastore)
+It is possible to use RDS as a datastore. Edit your configuration file for the control plane according to your RDS setting.
+```yaml
+  datastore:
+    type: MYSQL
+    config: 
+        url: root:password@tcp(arn_of_rds:3306) 
```

Comment:
This is an example, but it may be misleading that the pipecd organization allows running as the root user. What do you think?

Reply:
Yes, I think this expression leads misunderstanding that users must set root as the username, too.
I have changed.